incubator/java-openliberty/README.md

```diff
@@ -51,18 +53,21 @@ To get the generated password for project **my-project**, you can exec in the co
 
 So in the above example the password value would be: `2r1aquTO3VVUVON7kCDdzno`
 
-### OpenAPI
+#### OpenAPI
 
 The `mpOpenAPI` feature provides a set of Java interfaces and programming models that allow Java developers to natively produce OpenAPI v3 documents from their JAX-RS applications. This provides a standard interface for documenting and exposing RESTful APIs.
 
 OpenAPI endpoints:
 - http://localhost:9080/openapi (the RESTful APIs of the inventory service)
 - http://localhost:9080/openapi/ui (Swagger UI of the deployed APIs)
 
-### Junit 5
+#### Junit 5
 
 The default template uses JUnit 5. You may be used to JUnit 4, but here are some great reasons to make the switch https://developer.ibm.com/dwblog/2017/top-five-reasons-to-use-junit-5-java/
 
+### Kafka template
+
+The kafka template allows yoo to develop applications that connect to Kafka using the mpReactiveMessaging feature. For more information see the [kafka readme](templates/kafka/README.md)
```

Comment:
A few little things:
- Please fix up the typo: yoo / you.
- Can you backtick the `mpReactiveMessaging` so that it is shown as a code string.
- Please also add a comma after "For more information" and a full stop at the end of the sentence.

Reply:
Actually rather than `mpReactiveMessaging`, it might be better to state "MicroProfile Reactive Messaging" here for clarity and use the code string only in the Kafka readme.